scope.md

```diff
@@ -19,3 +21,20 @@ hyper-specific methods for one type of data to general, do-it-all packages
 (e.g. matplotlib). pyOpenSci accepts packages that are somewhere in between the 
 two. If you're interested in submitting your data visualization package, please 
 open a pre-submission inquiry on first.
+
+## Python package technical scope
```

Comment:
@arianesasso this is the scope document with the technical scope listed and it's included in the aims and scope page.